x-pack/plugins/cases/public/components/custom_fields/utils.test.ts

```diff
@@ -5,196 +5,288 @@
  * 2.0.
  */
 
-import { addOrReplaceCustomField } from './utils';
+import { addOrReplaceCustomField, customFieldSerializer } from './utils';
 import { customFieldsConfigurationMock, customFieldsMock } from '../../containers/mock';
+import type { CustomFieldConfiguration } from '../../../common/types/domain';
 import { CustomFieldTypes } from '../../../common/types/domain';
 import type { CaseUICustomField } from '../../../common/ui';
 
-describe('addOrReplaceCustomField ', () => {
-  beforeEach(() => {
-    jest.clearAllMocks();
-  });
+describe('utils ', () => {
```

Comment:
Make sure to select `Hide Whitespace` to only have the relevant changes in this file.